onnx/defs/rnn/defs.cc

```diff
@@ -71,6 +90,15 @@ allows to make spec more uniform.
     .Attr("num_layers", "Numbers of RNN layers in the stack, default 1",
           AttrType::INT)
     .Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT)
+    .Attr("activations", "A list of activation functions for the gates. Typical activation "
+          "functions are sigmoid and tanh",
+          AttrType::STRINGS)
+    .Attr("use_bias", "Use the bias tensors for the gates if 1. Default 1.",
```

Comment:
edit the comment for weights to describe that bias is optional in this case?

Reply:
Biases are still included in input(0) but not used if use_bias is false.